### PR TITLE
Help arguments

### PR DESCRIPTION
**Pull request by joshualyon7**

### CODE REVIEW

**Review by sbrugel — APPROVED** [review]

**Comment by joshualyon7** [comment]
Website view of what this looks like
image

**Comment by joshualyon7** [comment]
image

File: src/commands/info/help.ts [hunk 1]
```diff
@@ -38,6 +38,18 @@ export default class extends Command {
 				});
 			}
 
+			if (command.options) {
+				let concatOptions = '';
+				command.options.forEach(param => {
+					concatOptions += `**${param.name}** (${param.required ? 'required' : 'optional'}): ${param.description}\n`;
+				});
```
Comment (BenSegal855) [inline src/commands/info/help.ts:45]: This can be done with a .map() and a .join() if you want to avoid string concatenation and have it be a oneliner.

**Review by renross — APPROVED** [review]

**Review by bladetyr — APPROVED** [review]

**Review by sbrugel — APPROVED** [review]
almost forgot about this, sorry!

**Pull request merged**